src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/MSBuildSdkExtrasProject.cs

```diff
@@ -25,7 +25,7 @@ public MSBuildSdkExtrasProject ()
 		/// </summary>
 		public override bool ShouldRestorePackageReferences => true;
 
-		public string TargetFrameworks {
+		public new string TargetFrameworks {
 			get => GetProperty (nameof (TargetFrameworks));
```

Comment:
Can we remove `src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/MSBuildSdkExtrasProject.cs` completely? Instead of fixing the warnings.

It looks unused.